content/docs/command-reference/repro.md

```diff
@@ -178,8 +181,8 @@ up-to-date and only execute the final stage.
 
 - `--pull` - [pulls](/doc/command-reference/pull) dependencies and outputs
   involved in the stages being reproduced, if they are found in the
-  [default](/doc/command-reference/remote/default) remote storage. Note that it
-  checks the local run-cache too (available history of stage runs).
+  [default remote storage](/doc/command-reference/remote/default). Note that it
+  tries the local run-cache first.
 
   > Has no effect if combined with `--no-run-cache`.
```

Comment:
Question @pmrowla or @efiop (I forget who contributed this part):

If deps/outs are found in the remote but there's no local cache AND the `--no-run-cache` option is used, does it download them from the remote or not? I suspect that part here isn't exactly accurate. Thanks

Reply:
It will only be downloaded if `--pull` is specified.

Reply:
Right. But I meant if `--pull` is used WITH `--no-run-cache`, while there's no run-cache, `--pull` still does something? The way this is written now indicates that `--no-run-cache` somehow cancels all effects of `--pull`.

Reply:
Yes, `--pull` will still try to download using existing lockfiles, even if `--no-run-cache` is specified.

Reply:
OK then, corrected this a bit. Thanks